src/pages/playground.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import Layout from '@theme/Layout';
+import { useLocation } from 'react-router';
+import queryString from 'query-string';
+import { LabelText, Hero, Text } from '@algolia/ui-library';
+import DocSearch from '../components/DocSearch';
+import ErrorBoundary from '../components/ErrorBoundary';
+import algoliasearch from 'algoliasearch';
```

Comment:
We should load only the client with search capabilities, this one also handles indexing and is heavier.

```suggestion
import algoliasearch from 'algoliasearch/lite';
```

Reply:
will do